Project/Galagram/ViewModel/ViewModel/Admin/Comments/SingleViewModel.cs

```diff
@@ -29,14 +29,15 @@ public class SingleViewModel : SingleItemViewModelBase
         /// </exception>
         public SingleViewModel(DataAccess.Entities.Comment comment, bool isEditingEnabled)
             :base(shownEntity: comment, isWritingEnabled: isEditingEnabled)
-        {           
+        {
             ILookup<bool, string> groupedByLike = comment.Likes.ToLookup(c => c.IsLiked, u => u.User.NickName);
             this.likedUserNickname = groupedByLike[true].ToArray();
             this.disLikedUserNickname = groupedByLike[false].ToArray();
 
             // commands
-            deleteOrUpdateCommand = isEditingEnabled ? (ICommand) new Commands.Admin.UpdateCommand()
-                                                     : (ICommand) new Commands.Admin.DeleteCommand();
+            deleteOrUpdateCommand = isEditingEnabled ? (ICommand)new Commands.Admin.MultipleCommand(new ICommand[]
+                                                        { new Commands.Admin.Comment.Single.ValidateCommand(this), new Commands.Admin.UpdateCommand() })
```

Comment:
bad formating

Reply:
How would be right?

Reply:
{
   new Command()
   new Command()
}
the name seems to bo too long

Reply:
ok